tutorials/elixir-phoenix-on-google-app-engine.md

```diff
@@ -79,6 +79,70 @@ you already have an app to deploy, you may use it instead.
 4.  Visit [http://localhost:4000](http://localhost:4000) to see the Phoenix
     welcome screen running locally on your workstation.
 
+## Enable releases with Distillery
+
+Releases are the Elixir community's preferred way to package Elixir (and
+Erlang) applications for deployment. You will configure the
+[Distillery](https://github.com/bitwalker/distillery) tool to create releases
+for your app.
+
+**Note:** If you already have Distillery set up for your application, you can
```

Comment:
If you already have Distillery set up for your application, you can skip this section, but make sure you set `include_erts: true` in your `:prod` release configuration.

Reply:
fixed.